plugins/Goals/Menu.php

```diff
@@ -24,15 +24,13 @@ public function configureReportingMenu(MenuReporting $menu)
         $goals  = API::getInstance()->getGoals($idSite);
         $mainGoalMenu = 'Goals_Goals';
 
-        $linkToAddNewGoal = $this->urlForAction('addNewGoal', array(
-            'idGoal' => null
-        ));
+        $linkToManageGoals = $this->urlForAction('editGoals');
 
         $order = 1;
 
         if (count($goals) == 0) {
 
-            $menu->addItem($mainGoalMenu, '', $linkToAddNewGoal, 25);
+            $menu->addItem($mainGoalMenu, '', $linkToManageGoals, 25);
```

Comment:
the idea is to make it easy to users users to create their first goals since it's such a key feature of Piwik. linking to 'Add new goal' directly removes one extra click and puts users closer to better data insights using Piwik :+1: